allennlp/nn/util.py

```diff
@@ -656,6 +656,88 @@ def _get_combination(combination: str, tensors: List[torch.Tensor]) -> torch.Ten
             raise ConfigurationError("Invalid operation: " + operation)
 
 
+def weighted_tensor_combination(combination: str,
```

Comment:
It's confusing to call this `weighted_tensor_combination` - in the context of attention I think of the weighted combination as the `sum_i (a_i * h_i)` where a_i is the normalised attention distribution and h_i is the vector. Can you call it something else? `combine_tensors_and_multiply`?

Reply:
Yeah, I agree.